tips/totw/55.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Tip of the Week #55: Name Counting and unique_ptr
+layout: tips
+sidenav: side-nav-tips.html
+type: markdown
+---
+
+posted 2013-09-12
```

Comment:
Existing format is:

"Originally posted as totw/55 on Sep 12, 2013"

I do think we should keep the "Originally" ... at least, to emphasize its historical character.